charts/ingresslibchart/templates/_backendconfig.yaml

```diff
@@ -0,0 +1,22 @@
+{{- define "libchart.backendconfig.tpl" -}}
```

Comment:
Another naming consistency comment.

* Can the template prefix also match the chart name? Eg. `ingresslib.backendconfig` instead of `libchart.backendconfig`?
* Can the file be named `_backedconfig.tpl` instead of `_backedconfig.yaml`? (Since it's a helper)
* Can `.tpl` be removed from the template name (the `define` statement)? Just for the sake of following convention

Reply:
From my understanding .[tpl files are used when a template isn't rendered](https://helm.sh/docs/chart_template_guide/named_templates/). libchart.backendconfig.tpl is just the boilerplate, while [this](https://github.com/broadinstitute/terra-helm/blob/cde92ae3c01746d3a0d56a02d2b048004316763a/charts/ingresslibchart/templates/_backendconfig.yaml#L20) actually defines and outputs the chart. The .tpl extension also helps others who come to the code realize that it's not rending anything.[ Helm library documentation](https://helm.sh/docs/chart_template_guide/named_templates/) also follows this naming convention as well. Same applies for the file extension.

Reply:
Hey @knapii-developments , I think this is one reason why I'm confused!

I thought that library charts *only* included named templates, and don't render any manifests directly. I.e., the library chart includes a bunch of files like `_foo.tpl` and then the downstream charts use `include` or `template` to render the named templates from the library.

Reply:
So I guess I just need some clarification: is this chart intended to render things like BackendConfigs directly? Or should it only include named templates, which are then used in downstream charts like `agora`, with `include` statements?

Reply:
Let me know if this [comment](https://github.com/broadinstitute/terra-helm/pull/456#discussion_r694172569) makes sense

Reply:
Overall, IDK. I just followed the helm library documentation haha